### DISCUSSION TITLE
How to delete one of several vehicles from the Home screen?

### DISCUSSION THREAD

**Discussion by ManmooWall** [teslamate-org/teslamate#3918] [Q&A]
![tes](https://github.com/teslamate-org/teslamate/assets/142606407/33387522-8795-4281-9858-01989fab1736)
(I want to leave only the vehicles marked on the Red rectangle, and delete the vehicles marked on the Blue rectangle so that it is not recorded.)

I have two Tesla cars. One of them is used by other family member.
So I don't want that car to be recorded in Teslamate.
Is there any way?

https://docs.teslamate.org/docs/maintenance/manually_fixing_data/
Even if I delete the data with the 'Remove vehicle from the database' described on the above page, the data will be displayed again.
Perhaps the method described in the link is 'how to erase data that have been deleted from my current Tesla account'.
But I'm looking for 'how to stop recording one of the 2 cars currently on my Tesla account'.

**Comment by cwanja** [comment]
You would have to remove the Tesla from your account associated with TeslaMate.

**Reply by ManmooWall** [reply]
Looking for a way to stop logging for only one vehicle while maintaining vehicle registration on my Tesla account.

**Comment by mark3-dev** [comment]
I needed to do the exact same thing. I coded a way to enable / disable individual cars. 
Details are posted here on what I did https://github.com/teslamate-org/teslamate/discussions/1049

@cwanja How would I go about getting my PR back into the main repo?

**Reply by cwanja** [reply]
Create a pull request, include your changes and request it be merged. It will be reviewed @mark3-dev
